docker/storage/cassandra/install.sh

```diff
@@ -26,7 +26,9 @@ wget -qO- $APACHE_MIRROR/cassandra/$CASSANDRA_VERSION/apache-cassandra-$CASSANDR
   --strip=1
 
 # Merge in our custom configuration
-sed -i '/enable_user_defined_functions: false/cenable_user_defined_functions: true' conf/cassandra.yaml
+for feature in enable_user_defined_functions enable_sasi_indexes; do
```

Comment:
while this enables it, heh we still get server side warnings 'SASI indexes are experimental and are not recommended for production use.'